api/routes/me/put.js

```diff
@@ -2,7 +2,7 @@ const logger = require('../../logger')('me route put');
 const loggedIn = require('../../middleware').loggedIn;
 const deserializeUser = require('../../auth/serialization').deserializeUser;
 
-const editableFields = ['email', 'name', 'password', 'phone', 'position'];
+const editableFields = ['name', 'password', 'phone', 'position'];
```

Comment:
This is the only change necessary to make the fix.  Everything else is about tests and documentation.